### PR TITLE
Adding liveshare-dice-roller sample to teams toolkit sample repo

### PR DESCRIPTION
**Pull request by stephaniewhoo**
- Adding liveshare dice roller sample
- support local F5 and deployment in azure

### CODE REVIEW

File: live-share-dice-roller/.nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+12
```
Comment (HuihuiWu-Microsoft) [inline live-share-dice-roller/.nvmrc:1]: All existing samples in TTK supports Node.js v18, does this sample work with v18?

File: live-share-dice-roller/appPackage/manifest.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+    "version": "1.0.1",
+    "manifestVersion": "1.12",
```
Comment (HuihuiWu-Microsoft) [inline live-share-dice-roller/appPackage/manifest.json:4]: Better upgrade to v1.16 as other samples do.

File: live-share-dice-roller/appPackage/manifest.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.livesharediceroller",
+    "name": {
+        "short": "DiceRollerLiveShare",
```
Comment (HuihuiWu-Microsoft) [inline live-share-dice-roller/appPackage/manifest.json:8]: You can make it as 'DiceRollerLiveShare-${{TEAMSFX_ENV}}' so user can distinguish between local and remote.

File: live-share-dice-roller/teamsapp.yml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip # Relative path to this file. This is the path for built zip file.
+    # Output: following environment variable will be persisted in current environment's .env file.
+    # TEAMS_APP_ID: the id of Teams app
+  - uses: teamsApp/publishAppPackage # Publish the app to Teams Admin Center (https://admin.teams.microsoft.com/policies/manage-apps) for review and approval
+    with:
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+    writeToEnvironmentFile:
+      # Write the information of created resources into environment file for the specified environment variable(s).
+      publishedAppId: TEAMS_APP_PUBLISHED_APP_ID
+projectId: 836fd3d2-1be1-4ac5-8e0f-a7ec242f5dd2
```
Comment (HuihuiWu-Microsoft) [inline live-share-dice-roller/teamsapp.yml:87]: Please remove the field 'projectId'

File: live-share-dice-roller/.nvmrc (hunk 1 above)
Comment (stephaniewhoo) [reply live-share-dice-roller/.nvmrc:1]: yes

**Review by HuihuiWu-Microsoft — DISMISSED** [review]

File: live-share-dice-roller/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
+# Dice Roller - Live Share sample
+
+This repository contains a simple app that enables all connected clients to roll a dice and view the result. For a
+walkthrough of this example and how it works, check out the [tutorial documentation](https://aka.ms/fluid/tutorial).
+
+## Requirements
+
+Node 12.17+
```
Comment (jidddddd) [inline live-share-dice-roller/README.md:8]: Change to:
* [Node.js](https://nodejs.org/), supported versions: 16, 18
* An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
* [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [TeamsFx CLI](https://aka.ms/teamsfx-cli)

File: live-share-dice-roller/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- Click `Join` to join the meeting.
+- Select the tab in the bar, you will see a side panel tab in the meeting.
+
+## Deploy the app to Azure
+
+Deploy your project to Azure by following these steps:
+
+- Open Teams Toolkit in Visual Studio Code, and sign in your Azure account by clicking the `Sign in to Azure` in the `ACCOUNTS` section from sidebar.
+- After you signed in, select a subscription under your account. The Teams Toolkit will use this subscription to provision Azure resources to host you app.
+- Open the Teams Toolkit and click `Provision in the cloud` in the `DEVELOPMENT` section.
```
Comment (jidddddd) [inline live-share-dice-roller/README.md:26]: ...click `Provision` in the `LIFECYCLE` section.

File: live-share-dice-roller/README.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  - Alternatively open the command palette(Ctrl+Shift+P) and type: `Teams: Provision in the cloud` command.
```
Comment (jidddddd) [inline live-share-dice-roller/README.md:27]: ...type: `Teams: Provision`...

File: live-share-dice-roller/README.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+- Open the Teams Toolkit and click `Deploy to the cloud` in the `DEVELOPMENT` section.
```
Comment (jidddddd) [inline live-share-dice-roller/README.md:28]: ...click `Deploy` in the `LIFECYCLE` section.

File: live-share-dice-roller/README.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+  - Alternatively open the command palette(Ctrl+Shift+P) and type: `Teams: Deploy to the cloud` command.
```
Comment (jidddddd) [inline live-share-dice-roller/README.md:29]: ...type: `Teams: Deploy`...

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by jidddddd — APPROVED** [review]

**Comment by stephaniewhoo** [comment]
> @stephaniewhoo please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**